src/ui/views/Settings/settings.tsx

```diff
@@ -1003,6 +1004,24 @@ export const settings: {
 			}
 		},
 	},
+	{
+		category: "Schedule",
+		key: "tradeMatchingPercentage",
+		name: "Trade Matching Percentage",
+		godModeRequired: "existingLeagueOnly",
+		type: "int",
```

Comment:
Switch this to `intOrNull` so that blank is disabled, rather than 0. I think that's a little more straightforward.